charts/openebs/openebs-control-plane-pod-delete/experiment.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Kill all control plane pod and check if gets scheduled again
+kind: ChaosExperiment
+metadata:
+  labels:
+    litmuschaos.io/name: openebs
+  name: openebs-control-plane-pod-delete
+  version: 0.1.0
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      - apiGroups:
+          - ""
+          - "extensions"
+          - "apps"
+          - "batch"
+          - "litmuschaos.io"
+        resources:
+          - "pods"
```

Comment:
please add permission for powerfulseal LIB also, pls refer generic pod-delete experiment